play/psycopg3/README.md

```diff
@@ -0,0 +1,6 @@
+# Example using TAIL with psycopg3
+
+This example script shows how to create an asynchronous consumer of a Materialized View using the
+psycopg3 driver. You will need to follow the instructions for installing psycopg3 from source.
```

Comment:
Maybe link to the instructions? https://github.com/psycopg/psycopg3#installation

Reply:
Can do!